### ISSUE TITLE
event priority fails

### ISSUE THREAD

**Issue by SlimeDog** [SkriptLang/Skript#6288]
### Skript/Server Version

```markdown
> sk info
[17:18:44 INFO]: [Skript] Skript's aliases can be found here: https://github.com/SkriptLang/skript-aliases
[17:18:44 INFO]: [Skript] Skript's documentation can be found here: https://docs.skriptlang.org/
[17:18:44 INFO]: [Skript] Skript's tutorials can be found here: https://docs.skriptlang.org/tutorials
[17:18:44 INFO]: [Skript] Server Version: git-Paper-365 (MC: 1.20.4)
[17:18:44 INFO]: [Skript] Skript Version: 2.8.0-pre1 (skriptlang-github)
[17:18:44 INFO]: [Skript] Installed Skript Addons: None
[17:18:44 INFO]: [Skript] Installed dependencies: 
[17:18:44 INFO]: [Skript]  - Vault v1.7.3-b131
[17:18:44 INFO]: [Skript]  - WorldGuard v7.0.10-SNAPSHOT+2260-c7319f1
[17:18:44 INFO]: [Skript]  - Residence v5.1.4.1
```

### Bug Description

Scripts containing various time events fails to load. Examples may be found below.

### Expected Behavior

Clean parsing and execution

### Steps to Reproduce

- Update to Skript 2.8.0-pre1
- Start server
- Reload scripts
- Read console/log

### Errors or Screenshots

```
> sk reload scripts
[17:17:25 INFO]: [Skript] Reloading all scripts...
[17:17:25 INFO]: Line 19: (periodicals.sk)
[17:17:25 INFO]:     This event doesn't support event priority
[17:17:25 INFO]:     Line: at 06:00 in world "world_blackdog":
[17:17:25 INFO]:  
[17:17:25 INFO]: Line 21: (periodicals.sk)
[17:17:25 INFO]:     This event doesn't support event priority
[17:17:25 INFO]:     Line: at 18:00 in world "world_blackdog":
[17:17:25 INFO]:  
[17:17:25 INFO]: Line 24: (periodicals.sk)
[17:17:25 INFO]:     This event doesn't support event priority
[17:17:25 INFO]:     Line: at 06:00 in world "world_city":
[17:17:25 INFO]:  
[17:17:25 INFO]: Line 26: (periodicals.sk)
[17:17:25 INFO]:     This event doesn't support event priority
[17:17:25 INFO]:     Line: at 18:00 in world "world_city":
[17:17:25 INFO]:  
[17:17:25 INFO]: Line 29: (periodicals.sk)
[17:17:25 INFO]:     This event doesn't support event priority
[17:17:25 INFO]:     Line: at 06:00 in world "world_yellowdog":
[17:17:25 INFO]:  
[17:17:25 INFO]: Line 31: (periodicals.sk)
[17:17:25 INFO]:     This event doesn't support event priority
[17:17:25 INFO]:     Line: at 18:00 in world "world_yellowdog":
[17:17:25 INFO]:  
[17:17:25 INFO]: Line 68: (periodicals.sk)
[17:17:25 INFO]:     This event doesn't support event priority
[17:17:25 INFO]:     Line: every 1 minutes in "world_lobby":
[17:17:25 INFO]:  
[17:17:25 INFO]: Line 132: (periodicals.sk)
[17:17:25 INFO]:     This event doesn't support event priority
[17:17:25 INFO]:     Line: every 2 minutes in "world":
[17:17:25 INFO]:  
[17:17:25 INFO]: Line 178: (periodicals.sk)
[17:17:25 INFO]:     This event doesn't support event priority
[17:17:25 INFO]:     Line: every 2 minutes in "world_yellowdog":
[17:17:25 INFO]:  
[17:17:25 INFO]: Line 218: (periodicals.sk)
[17:17:25 INFO]:     This event doesn't support event priority
[17:17:25 INFO]:     Line: every 10 minutes in "world_lobby":
[17:17:25 INFO]:  
[17:17:25 INFO]: Line 55: (start-acid-rain.sk)
[17:17:25 INFO]:     This event doesn't support event priority
[17:17:25 INFO]:     Line: every 127 seconds in "world_whitedog":
[17:17:25 INFO]:  
[17:17:25 INFO]: Line 86: (start-acid-rain.sk)
[17:17:25 INFO]:     This event doesn't support event priority
[17:17:25 INFO]:     Line: every 107 seconds in "world_yellowdog":
```

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by APickledWalrus** [comment]
I believe this is caused by the data not being reset at https://github.com/SkriptLang/Skript/blob/38cc0fe26a4e88782d69da59206e30afeaad398e/src/main/java/ch/njol/skript/structures/StructEvent.java#L50

You *may* be able to resolve this issue temporarily by ensuring events with a custom priority come after events that don't support it.

**Comment by SlimeDog** [comment]
No worries. I'm just reporting what I find in scripts that used to work. I prefer not to change them unless absolutely necessary (ie, if there are design changes that render them unsupportable). These are test servers. I can wait for updates.

**Comment by SlimeDog** [comment]
This issue appears to be resolved in Skript 2.8.0-pre2. Thanks.
